source/agora/consensus/protocol/Nominator.d

```diff
@@ -686,10 +686,8 @@ extern(D):
                 auto data = deserializeFull!ConsensusData(candidate[]);
 
                 if (auto msg = this.ledger.validateConsensusData(data))
-                {
-                    log.error("combineCandidates(): Invalid consensus data: {}", msg);
-                    continue;
-                }
+                    assert(0, format!"combineCandidates: Invalid consensus data: %s"(
+                        msg));
                 else
```

Comment:
Please turn this into:

```D
log.info("combineCandidates: {}", slot_idx);
```

So just remove the `else { }` part.

Reply:
(unindent)

Reply:
Thanks, `else` removed.